src/Nox.Lib/Configuration/NoxUseOptions.cs

```diff
@@ -93,7 +93,8 @@ public void Configure(IApplicationBuilder builder)
         {
             appBuilder.UseMiddleware<VersionMiddleware>();
         });
-        builder.UseMiddleware<NoxApiMiddleware>().UseRouting();        
+        builder.UseMiddleware<RelatedEndpointsMiddleware>().UseRouting();
```

Comment:
the use routing is redundant applies to the IApplication Builder, to be more explicitly you can do:

`builder
.UseMiddleware<RelatedEndpointsMiddleware>()
.UseMiddleware<RelatedEndpointsMiddleware>()
.UseRouting();`

Reply:
updated